test/test_ops.py

```diff
@@ -722,6 +734,24 @@ def test_nms_cuda(self, iou, dtype=torch.float64):
             is_eq = torch.allclose(scores[r_cpu], scores[r_cuda.cpu()], rtol=tol, atol=tol)
         assert is_eq, err_msg.format(iou)
 
+    @needs_mps
+    @pytest.mark.parametrize("iou", (0.2, 0.5, 0.8))
```

Comment:
This looks like the same test as `test_nms_cuda` just above. Should we just parametrize it over devices instead of duplicating the test?

Something like this should work:

```py
    @pytest.mark.parametrize("device",
        (
            pytest.param("cuda", marks=pytest.mark.needs_cuda),
            pytest.param("mps", marks=pytest.mark.needs_mps),
        )
    )
```